### PR TITLE
py3-pip nak advisory CVE-2018-20225

### PR DESCRIPTION
**Pull request by pnasrat**
Same reason as py3.12-pip and below this is a contested advisory

### CODE REVIEW

**Comment by pnasrat** [comment]
Merge after https://github.com/wolfi-dev/os/pull/20318

**Review by cpanato — DISMISSED** [review]
thanks

**Comment by cpanato** [comment]
@pnasrat can you rebase and fix the conflict?

**Comment by pnasrat** [comment]
@cpanato done can you re-review

File: py3-pip.advisories.yaml [hunk 1]
```diff
@@ -9,6 +9,11 @@ advisories:
       - CVE-2018-20225
       - GHSA-7p5p-7qq5-cc86
     events:
+      - timestamp: 2024-06-20T14:20:25Z
+        type: false-positive-determination
+        data:
+          type: vulnerability-record-analysis-contested
+          note: This vulnerability is disputed, and the consensus in the security community is that this is intended behavior, not a security flaw.
```
Comment (cpanato) [inline py3-pip.advisories.yaml:16]: should be added after the detection event

**Review by cpanato — CHANGES_REQUESTED** [review]
a small comment

File: py3-pip.advisories.yaml (hunk 1 above)
Comment (pnasrat) [reply py3-pip.advisories.yaml]: @cpanato done!

**Pull request merged**